src/main/java/me/shedaniel/fiber2cloth/impl/Fiber2ClothImpl.java

```diff
@@ -276,12 +276,18 @@ private void initDefaultFunctionMap() {
             @SuppressWarnings("unchecked") ConfigLeaf<String> leaf = ((ConfigLeaf<String>) node);
             EnumSerializableType type = (EnumSerializableType) leaf.getConfigType();
             ClothSetting.EnumHandler.EnumDisplayOption displayOption = leaf.getAttributeValue(ClothAttributes.SUGGESTION_ENUM, ENUM_DISPLAY_TYPE).orElse(ClothSetting.EnumHandler.EnumDisplayOption.BUTTON);
+            String key = getFieldNameKey(leaf.getName()).getString();
             if (displayOption == ClothSetting.EnumHandler.EnumDisplayOption.BUTTON) {
                 return Collections.singletonList(configEntryBuilder
                         .startSelector(getFieldNameKey(leaf.getName()), type.getValidValues().toArray(new String[0]), leaf.getValue())
                         .setDefaultValue(leaf.getDefaultValue())
                         .setSaveConsumer(leaf::setValue)
                         .setErrorSupplier(v -> error(type, v))
+                        .setNameProvider((name) -> {
```

Comment:
I will change the implementation details on this, since you were supposed to implement Translatable in your enum, but I will merge this in a different way